test/mitmproxy/addons/test_disable_h2c.py

```diff
@@ -37,3 +37,5 @@ def test_prior_knowledge(self):
             a.request(f)
             assert not f.killable
             assert f.error.msg == flow.Error.KILLED_MESSAGE
+
+            a.request(f)
```

Comment:
Let's put this into a dedicated test please.

Reply:
Done :)